rationale.md

```diff
@@ -383,42 +427,192 @@ async create_query_task(
-  // Row limit (may override the limit in the saved query).
+  /**
+   * Row limit (may override the limit in the saved query).
+   */
   limit?: number
-  // Apply model-specified formatting to each result.
+  /**
+   * Apply model-specified formatting to each result.
+   */
   apply_formatting?: boolean
-  // Apply visualization options to results.
+  /**
+   * Apply visualization options to results.
+   */
   apply_vis?: boolean
-  // Get results from cache if available.
+  /**
+   * Get results from cache if available.
+   */
   cache?: boolean
-  // Render width for image formats.
+  /**
+   * Render width for image formats.
+   */
   image_width?: number
-  // Render height for image formats.
+  /**
+   * Render height for image formats.
+   */
   image_height?: number
-  // Generate drill links (only applicable to 'json_detail' format.
+  /**
+   * Generate drill links (only applicable to 'json_detail' format.
+   */
   generate_drill_links?: boolean
-  // Force use of production models even if the user is in development mode.
+  /**
+   * Force use of production models even if the user is in development mode.
+   */
   force_production?: boolean
-  // Retrieve any results from cache even if the results have expired.
+  /**
+   * Retrieve any results from cache even if the results have expired.
+   */
   cache_only?: boolean
-  // Prefix to use for drill links (url encoded).
+  /**
+   * Prefix to use for drill links (url encoded).
+   */
   path_prefix?: string
-  // Rebuild PDTS used in query.
+  /**
+   * Rebuild PDTS used in query.
+   */
   rebuild_pdts?: boolean
-  // Perform table calculations on query results
+  /**
+   * Perform table calculations on query results
+   */
   server_table_calcs?: boolean
-  // Requested fields
+  /**
+   * Requested fields
+   */
   fields?: string
 }
 ```
 
-### Missing features
+#### Methods with delimited list types
+
+As indicated in the [style values section of the OpenAPI specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#style-values), a list of values can be provided to a REST call in a variety of delimited formats. The Looker API uses "csv" formatting for some requests that receive simple lists, such as IDs.
+
+In Typescript, calling `toString()` on an array of numbers such as `[1,2,3]` would return `1, 2, 3`, which is fine for the Looker API methods expecting "csv-formatted" lists. However, in Python, the equivalent function returns `[1, 2, 3]` which would cause a parsing error by the Looker API endpoint.
+
+As indicated in the OpenAPI specification style values, other delimiters such as pipe (`|`), space, or tab characters could be expected by an API endpoint.
+
+To guarantee consistency in the serialization of "csv-formatted" array values, the code generator uses a **Delim** type for parameters that are specified as having a special delimiter. In Typescript, this type is called `DelimArray`. In Python, it's called `DelimSequence`.
```

Comment:
Any chance of coalescing these two terms for the same thing?  `DelimSequence` would work in TypeScript, yes?